guides/micronaut-data-jdbc-repository/metadata.json

```diff
@@ -4,14 +4,13 @@
   "title": "Access a database with Micronaut Data JDBC",
   "intro": "Learn how to access a database with Micronaut JDBC repositories.",
   "authors": ["Sergio del Amo"],
-  "tags": ["database", "micronaut-data", "jdbc", "h2", "flyway"],
+  "tags": ["database", "micronaut-data", "jdbc", "mysql", "flyway"],
```

Comment:
you don't need to specify a `mysql` tag. features are added as tags automatically.